src/main/java/com/wolox/training/models/Book.java

```diff
@@ -95,55 +98,55 @@ public String getAuthor() {
   }
 
   public void setAuthor(String author) {
-    this.author = author;
+    this.author = checkNotNull(author, Constant.NOT_FOUND_MESSAGE, "author");
   }
 
   public String getTitle() {
     return title;
   }
 
   public void setTitle(String title) {
-    this.title = title;
+    this.title = checkNotNull(title, Constant.NOT_FOUND_MESSAGE, "title");
   }
 
   public String getSubtitle() {
     return subtitle;
   }
 
   public void setSubtitle(String subtitle) {
-    this.subtitle = subtitle;
+    this.subtitle = checkNotNull(subtitle, Constant.NOT_FOUND_MESSAGE, "subtitle");
   }
 
   public String getPublisher() {
     return publisher;
   }
 
   public void setPublisher(String publisher) {
-    this.publisher = publisher;
+    this.publisher = checkNotNull(publisher, Constant.NOT_FOUND_MESSAGE, "publisher");
   }
 
   public String getYear() {
     return year;
   }
 
   public void setYear(String year) {
-    this.year = year;
+    this.year = checkNotNull(year, Constant.NOT_FOUND_MESSAGE, "year");
   }
 
   public Integer getPages() {
     return pages;
   }
 
   public void setPages(Integer pages) {
-    this.pages = pages;
+    this.pages = checkNotNull(pages, Constant.NOT_NULL_MESSAGE, "pages");
   }
```

Comment:
Add a check if the pages field is greater than 0.
Example:
Preconditions.checkArgument(pages>0,"It must be positive");

Remember to add this validation after checking if it is null.